### DISCUSSION TITLE
Urgent

### DISCUSSION THREAD

**Discussion by pbower** [ghiscoding/slickgrid-react#232] [General]
Hi @ghiscoding 

EDIT: I have reached out directly on LinkedIn for public thread reasons. If you're subscribed to this thread, kindly refer the original discussion email, or review LinkedIn.

Thanks,
Pete

**Comment by ghiscoding** [comment]
ok thanks I read your message and changed the key, I did not noticed that since it wasn't included in GitHub itself. So thanks for bringing this to my attention. 

I also pushed a new version and cleanup other unnecessary file as well from the npm publish, for example eslint config and other files that shouldn't be published, so thanks for pointing this out, it looks ok now and again I did change the key so it's all good now :)

**Comment by pbower** [comment]
No problem! Cheers.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
